### PR TITLE
Add Activity Start and End Date to Beneficiary Filters

### PR DESCRIPTION
**Pull request by tirgil**

### CODE REVIEW

**Comment by tirgil** [comment]
I changed the filter for dvr_activity instead of dvr_case_activity.

**Comment by tirgil** [comment]
@nursix done.

File: modules/templates/STL/config.py [hunk 1]
```diff
@@ -2323,7 +2323,15 @@ def custom_prep(r):
                             S3DateFilter("dvr_case.date",
                                          #label = T("Registration Date"),
                                          hidden = True,
-                                         ),
+                                        ),
+                            S3DateFilter("dvr_case_activity.activity_id$start_date",
+                                         label = T("Activity Start Date"),
+                                         hidden = True,
+                                        ),
```
Comment (nursix) [inline modules/templates/STL/config.py]: Closing parenthesis should be same indentation level as the enclosed list or block (see above).

Will fix after merge, but maybe you can next time? ;)

**Comment by nursix** [comment]
Btw - I think the list of filter options is too long, slowly it's getting rather difficult to navigate it (it's longer than the page, so you can't see the results without scrolling anymore).

We should consider to do something about it - do they really need all of these filter options?

File: modules/templates/STL/config.py (hunk 1 above)
Comment (tirgil) [inline modules/templates/STL/config.py]: I will :)

**Comment by tirgil** [comment]
Actually they said that they need all and they are very happy with these filters. :)

**Pull request merged**